lib/venue.rb

```diff
@@ -1,2 +1,37 @@
 class Venue
+  attr_reader :name, :capacity, :patrons
+
+  def initialize(name, capacity)
+    @name = name
+    @capacity = capacity
+    @patrons = []
+    @attendance = 0
+  end
+
+  def add_patron(patron)
+    @attendance += 1
+    @patrons << patron
+  end
+
+  def yell_at_patrons
+    yell_names = @patrons.map do |patron|
+      patron.upcase
+    end
+  end
+
+  def over_capacity?
+    if @attendance <= @capacity
```

Comment:
Currently, this method is failing. It is returning `true`.

So you have your `@patrons array` is there a way that you can see the `length` of that array (Hint: it will return an integer? And you have `@capacity` with is an integer. So can you compare those two things? 

Getting rid of your `@attendance`, you can find the capacity by doing something like:
```
@patrons.length > @capacity
```
Which will return to you a boolean